resources/dicts/relationship_events/normal_interactions/admiration/increase.json

```diff
@@ -1,311 +1,398 @@
+			"m_c remembers being like r_c and is impressed by {PRONOUN/r_c/poss} talent.",
+			"m_c thinks that r_c is so young, but so brave.",
+			"m_c can't wait to fight, mediate, or heal alongside r_c.",
+			"When speaking to r_c about {PRONOUN/r_c/poss} future, m_c can see a fiery passion.",
+			"m_c thinks that r_c will grow up to be an excellent member of the Clan.",
+			"m_c thinks r_c has a bright future ahead of {PRONOUN/r_c/object}."
+		],
+		"relationship_constraint": [
+			"admiration_30"
+		],
+		"main_status_constraint": [
+			"elder",
+			"warrior",
+			"mediator",
+			"medicine cat",
+			"deputy",
+			"leader"
+		],
+		"random_status_constraint": [
+			"kitten",
+			"apprentice",
+			"mediator apprentice",
+			"medicine cat apprentice"
+		]
+	},
+	{
+		"id": "admire_adult_odd_med1",
+		"interactions": [
+			"m_c feels like {PRONOUN/m_c/subject}'ll never accomplish as much as r_c.",
+			"m_c wonders if {PRONOUN/m_c/subject} can ever be as good as r_c.",
+			"m_c strives to catch up to r_c's success.",
+			"m_c can't imagine being as talented as r_c was in {PRONOUN/r_c/poss} youth.",
+			"m_c wants to tell stories as well as r_c.",
+			"m_c listens to r_c's fascinating life stories with envy and intrigue.",
+			"m_c hopes to leave behind a legacy like r_c."
+		],
+		"relationship_constraint": [
+			"admiration_30"
+		],
+		"main_status_constraint": [
+			"warrior",
+			"mediator",
+			"medicine cat",
+			"deputy",
+			"leader"
+		],
+		"random_status_constraint": [
+			"elder"
+		],
+		"also_influences": {
+			"jealousy": "increase"
+		}
+	},
+	{
+		"id": "admire_adult_odd_med1",
+		"interactions": [
+			"m_c thinks r_c is so much braver than {PRONOUN/m_c/object}.",
+			"m_c thinks r_c deserves what m_c is rewarded and more.",
+			"If it were a life-or-death situation, m_c thinks that r_c should be saved first.",
+			"m_c thinks about how to be more like r_c.",
+			"m_c tries to groom {PRONOUN/m_c/poss} pelt the same way r_c does.",
+			"m_c can barely keep up with r_c's talent.",
+			"m_c feels like r_c is more accomplished than {PRONOUN/m_c/object}."
+		],
+		"relationship_constraint": [
+			"admiration_60"
+		],
+		"main_status_constraint": [
+			"warrior",
+			"mediator",
+			"medicine cat",
+			"deputy",
+			"leader"
+		],
+		"random_status_constraint": [
+			"elder",
+			"warrior",
+			"mediator",
+			"medicine cat",
+			"deputy",
+			"leader"
+		],
+		"also_influences": {
+			"jealousy": "increase"
+		}
+	},
+	{
+		"id": "admire_inc_high1",
+		"intensity": "high",
+		"interactions": [
+			"m_c saw r_c show off a skill {PRONOUN/m_c/subject} never knew {PRONOUN/r_c/subject} had.",
+			"m_c is wowed by how bravely r_c fought the other day.",
+			"m_c is impressed by r_c's competence in any situation.",
+			"m_c is amazed at how skilled r_c really is.",
+			"m_c thinks the Clan's ancestors smile upon r_c.",
+			"m_c sees r_c save someone.",

```

Comment:
Remove this I think, we already have injury/death moon events specifically geared towards "savior" scenarios.